central/sensor/service/pipeline/virtualmachines/pipeline.go

```diff
@@ -0,0 +1,94 @@
+package virtualmachines
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	clusterDataStore "github.com/stackrox/rox/central/cluster/datastore"
+	"github.com/stackrox/rox/central/convert/internaltostorage"
+	countMetrics "github.com/stackrox/rox/central/metrics"
+	"github.com/stackrox/rox/central/sensor/service/common"
+	"github.com/stackrox/rox/central/sensor/service/pipeline"
+	"github.com/stackrox/rox/central/sensor/service/pipeline/reconciliation"
+	virtualMachineDataStore "github.com/stackrox/rox/central/virtualmachine/datastore"
+	"github.com/stackrox/rox/generated/internalapi/central"
+	"github.com/stackrox/rox/pkg/centralsensor"
+	"github.com/stackrox/rox/pkg/metrics"
+	"github.com/stackrox/rox/pkg/set"
+)
+
+var (
+	_ pipeline.Fragment = (*pipelineImpl)(nil)
+)
+
+func GetPipeline() pipeline.Fragment {
+	return newPipeline(clusterDataStore.Singleton(), virtualMachineDataStore.Singleton())
+}
+
+func newPipeline(
+	clusterStore clusterDataStore.DataStore,
+	virtualMachineStore virtualMachineDataStore.DataStore,
+) pipeline.Fragment {
+	return &pipelineImpl{
+		clusterStore:        clusterStore,
+		virtualMachineStore: virtualMachineStore,
+	}
+}
+
+type pipelineImpl struct {
+	clusterStore        clusterDataStore.DataStore
+	virtualMachineStore virtualMachineDataStore.DataStore
+}
+
+func (p *pipelineImpl) OnFinish(_ string) {}
+
+func (p *pipelineImpl) Capabilities() []centralsensor.CentralCapability { return nil }
```

Comment:
I think you should add here `centralsensor.VirtualMachinesSupported` as a capability.